### PR TITLE
Fix contact form bug

### PR DESCRIPTION
**Pull request by basil-kawak**
PTAL

### CODE REVIEW

**Review by gbilgedal — APPROVED** [review]

File: src/components/contact/index.css [hunk 1]
```diff
@@ -2,6 +2,14 @@
   width: 100%;
 }
 
+h1 {
```
Comment (louisrli) [inline src/components/contact/index.css:5]: do you want this to affect all p and h1 on every page?

File: src/components/contact/sections/ContactUs.jsx [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+    e.preventDefault();
+    emailjs
+      .sendForm(
+        "default_service",
+        "template_Uz0KFyDM",
+        e.target,
+        "user_31hCfU5FLOydTyY8iOnWx"
+      )
+      .then((result) => {
+        alert("Thank you, Your messeage have been sent");
```
Comment (louisrli) [inline src/components/contact/sections/ContactUs.jsx]: if you are planning to use this for the demo, please don't use an `alert`

Comment (louisrli) [reply src/components/contact/sections/ContactUs.jsx]: use setState or something and make it show a bootstrap box/alert or something like this

File: src/components/contact/index.css (hunk 1 above)
Comment (basil-kawak) [reply src/components/contact/index.css:5]: yes, we wanted to do that

**Comment by basil-kawak** [comment]
@louisrli PTAL

**Review by louisrli — APPROVED** [review]

**Pull request merged**